lib/plugins/aws/lib/updateStack.js

```diff
@@ -94,6 +94,24 @@ module.exports = {
       });
   },
 
+  validateStackTemplate() {
+    const compiledTemplate = _.cloneDeep(
+      this.serverless.service.provider.compiledCloudFormationTemplate);
+
+    this.serverless.cli.log('Validating Stack...');
+    const params = {
+      TemplateBody: JSON.stringify(compiledTemplate),
+    };
+
+    return this.provider.request(
+      'CloudFormation',
+      'validateTemplate',
+      params,
+      this.options.stage,
+      this.options.region
+    ).then((cfData) => this.monitorStack('validate', cfData));
```

Comment:
Exactly. According to the template validation function (http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CloudFormation.html#validateTemplate-property) the function returns an error if the validation fails (validation happens ONLINE/REMOTE). So monitorStack is inappropriate here and won't work, as there are no stack states changing at all. The solution would be to just check the result for an rejection and use this evaluation to decide if to proceed or not.